app/helpers/audits_helper.rb

```diff
@@ -269,7 +269,12 @@ def rebuild_audit_changes(audit)
     css_class_name = css_class_by_action(audit.action == 'destroy')
     audit.audited_changes.map do |name, change|
       next if change.nil? || change.to_s.empty?
-      next if name == 'template'
+      # update data for created template for better view
+      if name == 'template' && audit.action == 'create'
+        audit.audited_changes['template'] = ['', audit.audited_changes['template']]
+      else
+        next
```

Comment:
I believe there is a logical mistake.
The next was called for `name == 'template'`, now we are skipping everything apart of `name == 'template' && ...`. That's probably not what you've meant, right?
I believe we should still be skipping only if template, we should just add the changes if it's `create`. Considering we are touching the iterated array, wouldn't it be easier just to move the change before (or after) the loop and keep the loop itself untouched?

Reply:
Sorry, my bad.